pkg/kubelet/cm/cpumanager/state/state_file.go

```diff
@@ -52,13 +52,14 @@ func NewFileState(filePath string, policyName string) State {
 
 	if err := stateFile.tryRestoreState(); err != nil {
 		// could not restore state, init new state file
-		msg := fmt.Sprintf("[cpumanager] state file: unable to restore state from disk (%s)\n", err.Error()) +
-			"Panicking because we cannot guarantee sane CPU affinity for existing containers.\n" +
-			fmt.Sprintf("Please drain this node and delete the CPU manager state file \"%s\" before restarting Kubelet.", stateFile.stateFilePath)
-		panic(msg)
+		klog.Errorf("[cpumanager] state file: unable to restore state from disk (%v)"+
+			" We cannot guarantee sane CPU affinity for existing containers."+
+			" Please drain this node and delete the CPU manager state file \"%s\" before restarting Kubelet.",
+			err, stateFile.stateFilePath)
+		return nil, err
```

Comment:
Why remove the newlines here? It is usually easer to read if it is broken up, and forces it to jump out when this is the error.

I'm not 100% against the change, just curious about the reasoning to change it.

Reply:
It is just friendly for log collectors which are sensitive to newline. :)
 
Single-line error message may be easy for being collected with log prefix (level, time, file and line number, etc.).  
Multi-line one might lose prefix which contains that info. When these lines are sent to remote log db, It is sometimes hard to search and locate.